Procfile

```diff
@@ -1,2 +0,0 @@
-web: bundle exec unicorn -c ./config/unicorn.rb -p ${PORT:-3088}
-worker: bundle exec sidekiq -C ./config/sidekiq.yml
```

Comment:
Why is sidekiq being removed as well?

Reply:
It's more that this entire file is being removed, I think nothing uses it any more (but I'm willing to be corrected). Deployments use the Dockerfile, Kubernetes deployments use something else. I'll confirm this, though.

Reply:
Thread here: https://gds.slack.com/archives/CAB4Q3QBW/p1681806083832859